.github/workflows/ci.yml

```diff
@@ -119,6 +119,9 @@ jobs:
       - name: Poller
         run: make test-integration-poller
 
+      - name: Deployment modes
```

Comment:
I'm wondering if we could take the poller and deployment tests outside as a job to see if they can run at the same time

Reply:
Trying that here: https://github.com/grafana/tempo/pull/3928